lib/compat/wordpress-6.0/class-wp-webfonts.php

```diff
@@ -56,12 +64,30 @@ public function init() {
 	}
 
 	/**
-	 * Get the list of fonts.
+	 * Get the list of registered fonts.
+	 *
+	 * @return array
+	 */
+	public function get_registered_webfonts() {
+			return $this->registered_webfonts;
+	}
+
+	/**
+	 * Get the list of enqueued fonts.
+	 *
+	 * @return array
```

Comment:
```suggestion
	 * @return array[]
```

When populated, this returns an array of font-family arrays.